shell.nix

```diff
@@ -14,5 +14,6 @@ stdenv.mkDerivation {
     "disciplina=https://github.com/DisciplinaOU/disciplina/archive/master.tar.gz"
     "disciplina-faucet-frontend=https://github.com/DisciplinaOU/disciplina-faucet-frontend/archive/master.tar.gz"
     "disciplina-explorer-frontend=https://github.com/DisciplinaOU/disciplina-explorer-frontend/archive/master.tar.gz"
+    "disciplina-validatorcv=https://github.com/DisciplinaOU/disciplina-validatorcv/archive/release/0.1.0.tar.gz"
```

Comment:
let me merge release into master though? i think there wont be any fixes today on frontend

Reply:
Sure. When you do that, feel free to send a PR here as well.